### ISSUE TITLE
ECMASript grammar for CSharpSHarwell is in fact just CSharp

### ISSUE THREAD

**Issue by Vixorem** [antlr/grammars-v4#1668]
ECMASript grammar for CSharpSHarwell contains functions like LA, LT, that are not recognised with Sam Harwells antlr, thus they must be La and Lt. Grammar will biuld fine if we refactor function names.

**Comment by KvanTTT** [comment]
Have you tried [JavaScript](https://github.com/antlr/grammars-v4/tree/master/javascript/javascript) grammar instead?

**Comment by Vixorem** [comment]
@KvanTTT 
Yeah. I also replaced all the methods in ecma file so Harwell's antlr understands it. The thing was to turn the upper case methods into the ones with the first capital letter only.
